front-end/components/appInputComponents/ChatTextField.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState, ChangeEvent, KeyboardEvent } from 'react';
+import Box from '@mui/joy/Box';
+import Textarea from '@mui/joy/Textarea';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import SendIcon from '@mui/icons-material/Send';
+
+import axios from 'axios';
+
+export default function ChatTextField(){
+  
+    const submitStyle = {
+      position: 'fixed',
+      bottom: '80px',
+      right: '80px'
+    }
```

Comment:
To apply styles to a component, I typically prefer the `styled()` function. See example here: https://github.com/MecSimCalc/MecSimCalc-self-hosted/blob/main/front-end/components/AppCard.tsx#L21-L28